tests/custom-logger.test.js

```diff
@@ -85,3 +85,12 @@ test('Test generation custom logger default options', assert => {
 
   assert.end()
 })
+
+test('Test timestamp generation in milliseconds', assert => {
+  const [, timestampString] = timestampFunction().split(':')
+  const millisecondTimestampMagnitude = 1e12
```

Comment:
Instead of this, we could assert that `new Date(parseInt(timestampString, 10))` should be near this time. 

new Date accepts only ms, so if time should be second it fails (it would be in 1970 instead 2021)

Reply:
Now I convert the timestamp into date and verify that the year is greater than `2021`